src/main/resources/dataset/create-by-part.sql

```diff
@@ -8,6 +8,7 @@ select
   resources_and_parts.part_id,
   resources_and_parts.resource_id,
   resources_and_parts.resource_title,
+  resources_and_parts.question_id,

```

Comment:
The magic spreadsheet has this field, but we weren't using it in the frontend so I never added it. I ended up not using it for the dataset download because existing dataset downloads won't have it and I don't think it's particularly useful, but I wanted to add it to the query just in case for the future.